src/cron/personalizedDigest.ts

```diff
@@ -25,7 +29,17 @@ const cron: Cron = {
       .from(UserPersonalizedDigest, 'upd')
       .where('upd."preferredDay" = :nextPreferredDay', {
         nextPreferredDay,
-      });
+      })
+      .andWhere(
+        new Brackets((qb) => {
+          return qb
+            .where(`flags->>'sendType' IS NULL`)
+            .orWhere(`flags->>'sendType' = :sendType`, {
+              sendType: UserPersonalizedDigestSendType.weekly,
+            });
+        }),
+      );
```

Comment:
Only send weekly digest to old users and those who did not get allocated to daily digest experiment.